trunk/src/lsc/myloopdef.py

```diff
@@ -146,6 +157,65 @@ def run_getmag(imglist, _output='', _interactive=False, _show=False, _bin=1e-10,
     output_table['dmag'].format = '%.4f'
     if _output:
         output_table.write(_output, format='ascii')
+        if snex2_upload:
+            ### Make it snex2 readable
+            os.system('format_snex2.py -n ' + _output)
+            snex2_filename = splitext(_output)[0] + '_snex2.csv'
+
+            upload_extras = {
+                'reduction_type': 'manual',
+                'instrument': 'LCO'
+            }
+
+            if int(ftype)==1:
+                phot_type = raw_input('The default photometry type for non-difference imaging is PSF. If Aperture photometry was performed instead, please enter "Aperture". Otherwise press enter: ')
+                if phot_type.lower() == 'aperture':
+                    upload_extras['photometry_type'] = 'Aperture'
+                else:
+                    upload_extras['photometry_type'] = 'PSF'
+                    
+            elif int(ftype)==3:
+                phot_type = raw_input('The default photometry type for difference imaging is Aperture. If PSF photometry was performed instead, please enter "PSF". Otherwise press enter: ')
+                if phot_type.lower() == 'psf':
+                    upload_extras['photometry_type'] = 'PSF'
+                else:
+                    upload_extras['photometry_type'] = 'Aperture'
+                upload_extras['background_subtracted'] = True
+                if int(dtype)==0:
+                    upload_extras['subtraction_algorithm'] = 'Hotpants'
+                else:
+                    upload_extras['subtraction_algorithm'] = 'PyZOGY'
+
+                template_source = raw_input('Please enter the source of the template used to perform background subtraction (i.e. LCO or SDSS): ')
+                upload_extras['template_source'] = template_source
+
+            ### Prompt user for inputs for a few upload extras
+            reducer_group = raw_input('Please enter the group reducing this data (i.e. LCO, UC Davis, etc.): ')
+            if reducer_group!='LCO':
+                upload_extras['reducer_group'] = reducer_group
+
+            final_photometry = raw_input('Is this reduction final? [y/n]')
+            if final_photometry == 'y':
+                upload_extras['final_reduction'] = True
+            else:
+                upload_extras['final_reduction'] = False
```

Comment:
In most of the other question it looks like there is default behavior (which maybe happens somewhere else e.g. reducer_group isn't set to LCO explicitly by default and background_subtracted isn't set explicitly to False for filetype 1). To be consistent with that, it seems like this else statement is superfluous or the others should be specified

Reply:
It's true that some of the other "else" clauses are defined elsewhere in the code, but due to how we're selecting final vs not final photometry in the light curve plotting function in SNEx2, it's just much easier set `final_reduction` to True or False here. I did this intentionally because it mirrors how the upload extra values are set during manual dataproduct upload in SNEx2. Removing the else clause here would require rewriting some SNEx2 code in a few different places, which seems like too much of a pain for this minor change...